basic_flutter/lib/presentation/screens/splash_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({Key? key}) : super(key: key);
+
+  @override
+  _SplashScreenState createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(const Duration(seconds: 20), () {
```

Comment:
The timer duration is a bit too much, if you do 3 seconds here for a splash screen, it would be enough.